src/models/sighting/useSighting.js

```diff
@@ -1,13 +1,14 @@
 import { getSightingQueryKey } from '../../constants/queryKeys';
 import useFetch from '../../hooks/useFetch';
 
-export default function useSighting(sightingId) {
+export default function useSighting(sightingId, config = {}) {
   return useFetch({
     queryKey: getSightingQueryKey(sightingId),
     url: `/sightings/${sightingId}`,
     queryOptions: {
-      refetchInterval: 5000,
-      enabled: Boolean(sightingId),
+      retry: 1,
```

Comment:
I don't see how this is different than the default. If the query fails, then the failure count will be one. Because the failed query count has been met, the query will not retry.

Reply:
default is actually to retry three times.